public_officials/views.py

```diff
@@ -60,8 +94,15 @@ def state_legislators(request):
     return render(request, 'public-officials/state.html', {'state': state, 'senators': senators, 'representatives': representatives})
 
 def bills_index(request):
-    bill_service = BillService()
-    bills = bill_service.get_recent_bills()
+    cache_key = 'recent_bills'
+    cache_time = 1800
```

Comment:
Good call expiring this one every 30 mins since this one updates more frequently.